node_modules/oae-core/filepreview/filepreview.html

```diff
@@ -54,6 +54,32 @@
     </div>
 --></div>
 
+<!-- VIDEO PREVIEW -->
+<div id="filepreview-video-template"><!--
+    <div id="filepreview-video-container" class="text-center">
+        {if content.previews && content.previews.video}
+            {var url = content.previews.video.replace('remote:', '')}
+            {var mime = 'video/mp4'}
+        {else}
+            {var url = content.downloadPath}
+            {var mime = content.mime}
+        {/if}
+
+        {if content.previews && content.previews.height} 
+            {var height = content.previews.height}
```

Comment:
Cool - I'll also change the Zencoder settings in the Hilary pr to limit the height of video to a max of 600px (no point making it bigger if we won't display the excess).